NuKeeper.Git/GitCmdDriver.cs

```diff
@@ -102,5 +113,11 @@ private Uri CreateCredentialsUri(Uri pullEndpoint, GitUsernamePasswordCredential
 
             return new UriBuilder(pullEndpoint) { UserName = gitCredentials.Username, Password = gitCredentials.Password }.Uri;
         }
+
+        public Task<IReadOnlyCollection<string>> GetNewCommitMessages(string baseBranchName, string headBranchName)
+        {
+            // cmd should be something like: $"log Branch-Name-Template..master --decorate=short";
+            throw new NotImplementedException();
```

Comment:
This needs to be implemented?

Reply:
Like a lot more of the driver, it looks "unfinished". I just left a hint, but can try to implement this part.

Reply:
Implemented some parts of the driver that where needed for the test, and added tests. Still looks unfinished to me, but the basics now work